spydra/src/main/java/com/spotify/spydra/submitter/api/DynamicSubmitter.java

```diff
@@ -155,6 +157,9 @@ boolean createNewCluster(SpydraArgument arguments, DataprocAPI dataprocAPI)
     } else {
       createArguments = arguments;
     }
+
+    arguments.addOption(createArguments.cluster.options, SpydraArgument.OPTION_LABELS, SPYDRA_CLUSTER_LABEL + "=1");
```

Comment:
What is the semantic meaning of 1 here?

Reply:
Just to mark it as a spydra cluster, so to take it into consideration.

Reply:
Ok.